test/unit/files/test_http.py

```diff
@@ -87,7 +87,7 @@ def test_file_source_ftp_url():
 
     def check_generic_headers(request, **kwargs):
         assert not request.headers
-        response: Any = io.StringIO("This is ftp.gnu.org, the FTP server of the the GNU project.")
+        response: Any = io.StringIO("This is ftp.gnu.org, the FTP server of the GNU project.")
```

Comment:
It's just the test response so it doesn't matter as long as it matches up with below, but it's probably worth keeping this in line with the other bit.

Reply:
Yes, I think so, just to be consistent with the ftp.gnu.org response.